lib/routes/subswap/swap/swap_page.dart

```diff
@@ -21,70 +24,56 @@ class SwapPage extends StatefulWidget {
 
 class SwapPageState extends State<SwapPage> {
   final BreezBridge breezLib = ServiceInjector().breezLib;
-  SwapInfo? swap;
+  SwapInfo? swapInProgress;
+  SwapInfo? swapUnused;
   String? bitcoinAddress;
   String? errorMessage;
 
-  @override
-  void didChangeDependencies() {
-    getBTCAddress();
-    super.didChangeDependencies();
-  }
-
-  void getBTCAddress() async {
-    try {
-      swap = (await breezLib.receiveOnchain());
-    } catch (e) {
-      errorMessage = extractExceptionMessage(e);
-    } finally {
-      setState(() {});
-    }
-  }
-
   @override
   Widget build(BuildContext context) {
-    final texts = context.texts();
-    final bool isLoading = swap == null && errorMessage == null;
+    return BlocBuilder<SwapInBloc, SwapInState>(builder: (context, swapState) {
+      final SwapInBloc swapInBloc = context.read();
```

Comment:
Please define expected type `<SwapInBloc>`

Reply:
Why? It is inferred from the type.

Reply:
My bad, didn't notice you defined the type here.
I'll go over the usages of this context extension and pick a single convention to help with search indexing.